examples/nlp/language_modeling/tuning/conf/megatron_gpt_adapter_tuning_config.yaml

```diff
@@ -103,9 +103,12 @@ model:
     answer_field: "target"
 
   adapter_tuning:
+    type: 'parallel'
```

Comment:
Can you add a list of possible args here? What does parallel mean?

Reply:
If it is meant to differentiate between megatron adapters and vanilla pytorch adapters I would use that nomenclature instead.

Reply:
parallel was used signify using Column/Row Linear Parallel instead of nn.Linear. Its not set to 'linear_adapter' or 'parallel_adapter'